pkg/flowkit/services/project.go

```diff
@@ -160,7 +161,7 @@ func (p *Project) Deploy(network string, update bool) ([]*contracts.Contract, er
 			return nil, err
 		}
 
-		targetAccount, err := p.state.Accounts().ByAddress(contract.Target())
+		targetAccount, err := p.state.Accounts().ByAddressAndAccountName(contract.Target(), contract.AccountName())
```

Comment:
now that i think about it, we can just use existing function of ByName instead of filtering by both address and account name

Reply:
I'm now wondering the same. I think that accounts must be unique by their names (restriction should be added to the config module) and hence adding the address is unnecessary. I believe the code would be nicer if it just gets the accounts by name.

Reply:
resolved